pkg/workflows/sdk/v2/const.go

```diff
@@ -0,0 +1,3 @@
+package sdk
+
+const IdLen = 24
```

Comment:
Why 24? Add a comment with explanation?

Reply:
I don't remember why I picked 24... I'll change it to be 16 bytes so it's the size of a UUID.